plugins/katello/developers.md

```diff
@@ -326,19 +251,19 @@ tito release koji
 /usr/local/bin/katello-mash-split-X.Y.py
 ```
 
-### Step 11: Release X.Y Release Candidate RPMs
+### Step 12: Release X.Y Release Candidate RPMs
 
 For this step we'll need to update the Jenkins job that pushes RPMs to fedorapeople and run the job. Go to `http://ci.theforeman.org/view/Katello%20Pipeline/job/release_push_rpm_katello/configure` and add X.Y to the list of RELEASE choices and save the configuration. Now head to `http://ci.theforeman.org/view/Katello%20Pipeline/job/release_push_rpm_katello/build?delay=0sec`, choose the X.Y version you just added and click `Build`. This will run a script that copies the RPMs from Koji to fedorapeople.
 
-### Step 12: Update forklift
+### Step 13: Update forklift
 
 We maintain a deployment tool, forklift, for easily testing and deploying Katello versions. You'll want to clone the repository (https://github.com/theforeman/forklift) and update the following:
 
 1. Edit the README to indicate the new version and the OSes it works with.
 1. Update the foreman version mappings - https://github.com/theforeman/forklift/blob/master/config/versions.yaml
 1. Add new boxes for X.Y - https://github.com/theforeman/forklift/blob/master/config/base_boxes.yaml
 
-### Step 13: Test
+### Step 14: Test
```

Comment:
Should add a note about making the pipeline playbooks and using those to run a pre-flight test of each RC and release?